zcl-builtin/shared/schema/zcl.xsd

```diff
@@ -479,6 +479,7 @@ This schema describes the format of the XML files, that describe the ZCL specifi
       <xs:attribute name="array" type="xs:boolean"/>
       <xs:attribute name="isNullable" type="xs:boolean"/>
       <xs:attribute name="mustUseTimedWrite" type="xs:boolean"/>
+      <xs:attribute name="apiMaturity" type="zclApiMaturity"/>
```

Comment:
You need to run the make-schema-diagram to update the schema diagram for this as well.

Reply:
Done. I also added the make script update to the PR.